### PR TITLE
Add images preview in edit mode plus add keydown cleanup

### PR DESCRIPTION
**Pull request by ivictbor**
Hey @Ionaru , 
thanks for taking a fork of this and continue working on it! Our team needs some lightweight embeddable markdown editor with good visual capabilities so we decided to take a fork from EasyMDE and will return the features to upstream if you have time to review/discuss it and will be ready to move in the same direction.

I started with a visualisation of images.
Here is a preview how it looks:

Same feature was in (https://stackedit.io/app), but that editor is very heavy/not embeddable and does not work on mobile when EasyMDE is neat and suites most needs.

My planned next actions: (optional setting for buttons which will be put on top of images - this would allow easily attach image editor/delete an image).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,6 +73,8 @@ var easyMDE = new EasyMDE();
 </script>
 ```
 
+If you need to remove installed listeners (when editor not needed anymore), call `easyMDE.cleanup()`
+
```
Comment (Ionaru) [inline README.md:77]: The placement of the documentation is a bit odd, in between the parts describing how to load the editor. Maybe move this to https://github.com/Ionaru/easy-markdown-editor#removing-easymde-from-text-area ?

File: src/js/easymde.js [hunk 2]
```diff
@@ -2048,7 +2049,8 @@ EasyMDE.prototype.render = function (el) {
     if (options.status !== false) {
         this.gui.statusbar = this.createStatusbar();
     }
-    if (options.autosave != undefined && options.autosave.enabled === true) {
+    var autosaveEnabled = options.autosave != undefined && options.autosave.enabled === true;
+    if (autosaveEnabled) {
```
Comment (Ionaru) [inline src/js/easymde.js:2053]: What's the purpose of this change?

Comment (ivictbor) [reply src/js/easymde.js]: @Ionaru well basically I reused change listener, so reused this check, though you ar right,  rolled back now

File: README.md (hunk 1 above)
Comment (ivictbor) [reply README.md]: @Ionaru agree, thx, done

**Review by Ionaru — APPROVED** [review]
Thank you for the contribution!
